getGit.tex

```diff
@@ -154,23 +185,12 @@ \section{What, Why, and How?}
 \begin{frame}
     \frametitle{Git and Operating Systems}
     \begin{itemize}
-        \item{Windows doesn't git well.}
-        \item{Linux install through package manager}
-        \item{Mac: ???}
-    \end{itemize}
-    \vspace{1cm} % generate some space between title and content
-\end{frame}
-
-\begin{frame}
-    \frametitle{Git and Operating Systems}
-    \begin{itemize}
-        \item{Windows doesn't git well.}
-        \item{Linux install through package manager}
+        \item{Windows: Install gitbash.}
+        \item{Linux: Install through package manager.}
         \item{Mac: ???}
     \end{itemize}
     \vspace{1cm} % generate some space between title and content
 \end{frame}
-\section{Getting Started with git}
```

Comment:
Why did you drop this?

Reply:
I think I was just messing with deleting stuff to see what effect it had on the formatting, and I forgot to add it back. I fixed it now my bad.